### ISSUE TITLE
Integrate axe-core into our e2e tests

### ISSUE THREAD

**Issue by terrazoon** [GSA/notifications-admin#1926]
It looks like axe-core can easily be added to our e2e tests, giving us at least some level of automated accessibility testing.   Write a new axe-core e2e test that just movies from page to page looking for violations.
